pyiron_base/jobs/job/runfunction.py

```diff
@@ -1,15 +1,19 @@
 # coding: utf-8
 # Copyright (c) Max-Planck-Institut für Eisenforschung GmbH - Computational Materials Design (CM) Department
 # Distributed under the terms of "New BSD License", see the LICENSE file.
+from concurrent.futures import ProcessPoolExecutor
 from datetime import datetime
 import multiprocessing
 import os
 import posixpath
 import subprocess
 
+from jinja2 import Template
```

Comment:
This needs to be added to `setup.py` and `.ci_support/environment.yml`, or? I have it on my machine, but it's not in [the standard library](https://docs.python.org/3/library/), so I think we should specify it.

Reply:
`jinja2` is required for `flux` so I move it to the other pull request.